src/main/scala/fi/aalto/cs/apluscourses/intellij/actions/REPLAction.scala

```diff
@@ -0,0 +1,25 @@
+package fi.aalto.cs.apluscourses.intellij.actions
+
+import com.intellij.openapi.actionSystem.{AnActionEvent, CommonDataKeys}
+import com.intellij.openapi.module.{ModuleManager, ModuleUtilCore}
+import org.jetbrains.plugins.scala.console.actions.RunConsoleAction
+
+class REPLAction extends RunConsoleAction {
+
+  override def actionPerformed(e: AnActionEvent): Unit = {
```

Comment:
what happens in general:
we point our plugin to the new action class responsible for the processing. when the actual event is fired, it gets intercepted here, we can adjust the load of the event as we need and then pass it further to the actual implementation (of Scala plugin). with this approach, at least based on simple testing, the impact is minimal as well as dependency, we just reuse...

Reply:
This seems like a great approach! I assume we would insert some module selection stuff here before calling the Scala plugin's `actionPerformed`?

This way we avoid having to implement anything directly related to the REPL.